components/config/src/config/link_checker.rs

```diff
@@ -1,10 +1,48 @@
+use console;
 use serde::{Deserialize, Serialize};
 
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
+pub enum LinkCheckerLevel {
+    #[serde(rename = "error")]
+    Error,
+    #[serde(rename = "warn")]
+    Warn,
+}
+
+impl Default for LinkCheckerLevel {
+    fn default() -> Self {
+        Self::Error
+    }
+}
+
+impl LinkCheckerLevel {
+    pub fn log(&self, msg: String) {
+        match self {
+            LinkCheckerLevel::Error => {
+                console::error(format!("{}{}", self.log_prefix(), msg).as_str())
+            }
+            LinkCheckerLevel::Warn => {
+                console::warn(format!("{}{}", self.log_prefix(), msg).as_str())
+            }
+        }
+    }
```

Comment:
hm having the config depend on the console module is not ideal, maybe it was a bad idea

Reply:
What issue do you see with config depending on console?

Reply:
Gotcha, what would you suggest then for logging warnings?  Should I duplicate the terminal colorization code into link_checker to avoid having a dependency on `console`?  Or not colorize warnings at all?  I'm open to whatever.

Reply:
I think https://github.com/getzola/zola/pull/1848#discussion_r866069170 will resolve that problem.  I'll give it a try.